extensions/packageManager/src/client/feedModal.tsx

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { PrimaryButton, DialogFooter, DefaultButton } from 'office-ui-fabric-react';
+import { useState, useEffect, Fragment } from 'react';
+import {
+  CheckboxVisibility,
+  DetailsList,
+  DetailsListLayoutMode,
+  Selection,
+  SelectionMode,
+} from 'office-ui-fabric-react/lib/DetailsList';
+import { IconButton, ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { useApplicationApi } from '@bfc/extension-client';
+
+import { DialogWrapper, DialogTypes } from '@bfc/ui-shared';
+import formatMessage from 'format-message';
+import { v4 as uuid } from 'uuid';
+
+import { PackageSourceFeed } from './Library';
+
+const uitext = {
+  title: formatMessage('Feeds'),
```

Comment:
formatMessage should be called from within the component to take effect, either inline the texts where they are used or make this a function.